Migrations/20220626003117_add_qrcode_as_binary_and_qrcodetimestamp_to_room.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace CoronaCheckIn.Migrations
+{
+    public partial class add_qrcode_as_binary_and_qrcodetimestamp_to_room : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "QrCode",
+                table: "Rooms");
+
+            migrationBuilder.DropColumn(
+                name: "QrCodeTimestamp",
+                table: "Rooms");
```

Comment:
Die hier sind zu viel, da du schon in Migrations/20220625203732_temporarily_remove_qrcode_from_room.cs dropst